docker-for-mac/troubleshoot.md

```diff
@@ -64,8 +64,7 @@ Once the diagnostics are available, you can upload them and obtain a
 team. For more information on our policy regarding personal data, see
 [how is personal data handled in Docker Desktop](faqs.md#how-is-personal-data-handled-in-docker-desktop).
 
-![Diagnostics & Feedback with
-ID](images/diagnose-feedback-id.png){:width="600px"}
+![Diagnostics & Feedback with ID](images/diagnose-feedback-id.png){:width="600px"}
```

Comment:
Looks like images also run into the bug with the "jekyll-relative-links" plugin, so if relative paths are used, we can't wrap them